cdap-unit-test/src/main/java/co/cask/cdap/test/internal/ApplicationManagerFactory.java

```diff
@@ -25,5 +25,5 @@
  */
 public interface ApplicationManagerFactory {
 
-  ApplicationManager create(@Assisted("applicationId") Id.Application applicationId);
+  ApplicationManager create(@Assisted("applicationId") ApplicationId applicationId);
```

Comment:
seems fine to me. Users can technically see this class, but can't (and shouldn't) do anything with it.